src/rollup/BridgeCreator.sol

```diff
@@ -20,177 +20,106 @@ import "@openzeppelin/contracts/access/Ownable.sol";
 import "@openzeppelin/contracts/proxy/transparent/ProxyAdmin.sol";
 
 contract BridgeCreator is Ownable {
-    ContractTemplates public ethBasedTemplates;
-    ContractERC20Templates public erc20BasedTemplates;
+    BridgeContracts public ethBasedTemplates;
+    BridgeContracts public erc20BasedTemplates;
```

Comment:
Nice refactor of BridgeCreator, can you also reflect the changes to Foundry unit tests `BridgeCreatorTest` and `RollupCreatorTest`? They are part of the CI loop

Reply:
updated the unit tests